geomstats/geometry/matrices.py

```diff
@@ -26,34 +28,57 @@ def belongs(self, point):
         return mat_dim_1 == self.m & mat_dim_2 == self.n
 
     @staticmethod
-    def mul(a, b):
-        """Return the product of matrices a and b.
+    def equal(mat_a, mat_b, atol=TOLERANCE):
+        """
+        Test if matrices a and b are close.
 
         Parameters
         ----------
-        a : array-like, shape=[n_samples, dim1, dim2]
-        b : array-like, shape=[n_samples, dim2, dim3]
+        mat_a : array-like, shape=[n_samples, dim1, dim2]
+        mat_b : array-like, shape=[n_samples, dim2, dim3]
 
         Returns
         -------
-        c : array-like, shape=[n_samples, dim1, dim3]
+        eq : array-like boolean, shape=[n_samples]
```

Comment:
Rebase on master, so that we do not see the changes of the other PR.

Reply:
think I managed :)